gen3/bin/kube-setup-secrets.sh

```diff
@@ -102,7 +102,13 @@ if [[ (! -f "$configmapsFlagFile") || $(stat -c %Y "$configmapsFlagFile") -lt "$
   gen3 gitops configmaps
   touch "$configmapsFlagFile"
 fi
-
+# ssjdispatcher
+if ! g3kubectl get secrets/ssjdispatcher-creds > /dev/null 2>&1; then
```

Comment:
All of the other `-creds` secrets come from `creds.json` - just add an `ssjdispatcher` block to `creds.json` - do not introduce yet another secrets file

Reply:
Fixed in a55945ffa92f01cdd9dc040a8fa35656ee373220